presto-hive/src/main/java/com/facebook/presto/hive/StoragePartitionLoader.java

```diff
@@ -93,6 +94,7 @@
 import static java.lang.Math.max;
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
+import static org.apache.hadoop.mapreduce.lib.input.FileInputFormat.SPLIT_MINSIZE;
```

Comment:
This feels like a large dependency to add just for a constant string. Consider using the literal value instead.